### PR TITLE
Add documentation for Windows

### PR DESCRIPTION
**Pull request by activars**
- Improve the configuration documentation for Windows setup 
- Give real example for bash and PowerShell setup

### CODE REVIEW

**Comment by activars** [comment]
looks like a build to failure, not related to the commit. How can i trigger a retest without a commit?

**Comment by activars** [comment]
anyone to review this?

File: app/config-default.js [hunk 1]
```diff
@@ -68,11 +68,18 @@ module.exports = {
...
     // if left empty, your system's login shell will be used by default
-    // make sure to use a full path if the binary name doesn't work
-    // (e.g `C:\\Windows\\System32\\bash.exe` instead of just `bash.exe`)
-    // if you're using powershell, make sure to remove the `--login` below
+    //
+    // Windows
+    // - Make sure to use a full path if the binary name doesn't work
+    // - Remove `--login` in shellArgs
+    //
+    // Bash on Windows
```
Comment (DanielGGordon) [inline app/config-default.js]: Is this a reference to the new WSL bash shell? That's not too clear here.

Comment (activars) [inline app/config-default.js]: Yes indeed. I will add additional text to clarify.

Comment (activars) [inline app/config-default.js]: and squash the commit..

**Comment by albinekb** [comment]
closed & reopened to trigger builds

File: app/config-default.js [hunk 2]
```diff
@@ -70,9 +70,16 @@ module.exports = {
 
     // the shell to run when spawning a new session (i.e. /usr/local/bin/fish)
     // if left empty, your system's login shell will be used by default
-    // make sure to use a full path if the binary name doesn't work
-    // (e.g `C:\\Windows\\System32\\bash.exe` instead of just `bash.exe`)
-    // if you're using powershell, make sure to remove the `--login` below
+    //
```
Comment (chabou) [inline app/config-default.js]: There's a trailing space here. (See tests)
@activars can you fix it?

File: app/config-default.js [hunk 3]
```diff
@@ -70,9 +70,16 @@ module.exports = {
...
+    // Windows
+    // - Make sure to use a full path if the binary name doesn't work
+    // - Remove `--login` in shellArgs
+    //
```
Comment (chabou) [inline app/config-default.js]: There's a trailing space here. (See tests)
@activars can you fix it?

**Pull request merged**